### ISSUE TITLE
Let me know

### ISSUE THREAD

**Issue by nw** [WebDevBootcamp/archiVe#28]
@patrickmulcrone - nice work on the organization. Also impressed that your working from a fork and issuing pull requests.

Is anyone struggling with merging or pushing commits? Happy to help.

**Comment by nw** [comment]
Also seem like there are a lot of tickets are open. Maybe some housekeeping is in order. Narrow down or consolidate the issues that are open. Maybe mark some as "nice to have" or "won't fix".

**Comment by selinaross** [comment]
Nate, 

Thanks for your feedback. I'll let you know if I have trouble pushing and/or merging and will clean up the issues.

**Comment by patrickmulcrone** [comment]
Thanks for feedback and readme mark down! It looks great!

**Comment by dlpeters** [comment]
Hey patrick, from nate's video do you clone from https://github.com/WebDevBootcamp/archiVe?  I've been trying to get it to work for an hour ... I think it might be my sourcetree cuz I get
fatal open /dev/null or dup failed  (really helpful error message NOT).  I have verified all my directories are there    (I got stuck using my unreliable PC instead of the MAC which my son has taken from the house .. Argh!)

I downloaded the zip file and can work using webstorm. looks like the zip took the code from master (i tried to clone from there already) I can work on loading the real book images and doing the actual search ... I don't see Yvonne's code to do the search in gitHub. I'm assuming that's a good place to work. I will worry about merging when I get this "cloning" problem worked out.

**Comment by selinaross** [comment]
Hey Deb - 

Did you fork the repo in github and clone it in sourcetree?

**Comment by nw** [comment]
@dlpeters - I would start from scratch if your having trouble.
- open sourcetree.
- click "new repository"
- select "clone from url"
- source url is: `https://github.com/WebDevBootcamp/archiVe.git`
- make sure you set your destination path and name correctly.

If your selecting a new folder make sure the setting stick. I have seen sourcetree revert changes here for some odd reason.

Open the repo view in sourcetree. You can right click on the branches label and `create a new branch`. Once you have done that. You should see `master` and your new branch in bold. Double clicking will switch branches.

All you need to do is make your changes and commit. We can worry about merging in class on monday.

Let me know if you can't past the setup and we can figure something out.

**Comment by dlpeters** [comment]
Thanks Selina and Nate!  Darn, I thought it was going to work. followed Nate's steps again (PC interface is slightly different than MAc).   I notice the clipboard pastes dlpeters@github.com (using @ sign .. I also tried replacing with . )  Is that correct?

![capture](https://cloud.githubusercontent.com/assets/2429555/7667179/8ea3fb56-fbbc-11e4-87cc-b54ac97c8cc4.JPG)

I can see the directories so I know they are there

**Comment by nw** [comment]
Yeah this is what I was referring to.

You need to manually create the folder you want the repo to be at.

Open up `C:\Users\Deb\WebstormProjects\` in windows explorer. Then create a new folder in there called `archiVe1`. Then try again.

**Comment by dlpeters** [comment]
yes, I have that

**Comment by nw** [comment]
hmm.... are you sure it's not `archiVe` instead of `archiVe1`?

Is the directory empty?

**Comment by dlpeters** [comment]
I made archiVe1 using webstorm it is empty.  That's OK.  I'll work on the unzipped files for awhile. When I get my MAC back I will try it on there ... things seem to work better on MAC. I've just been having problems with this PC so, who knows. 
One last question .. so I did what Selina said forked from weblab into dlpeters and then tried to clone from dlpeters ... that's right isn't it? Or, do I just clone from weblab to source tree?  (I've tried both)
